virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -232,7 +254,7 @@ If your worker nodes have different CPUs, live migration failures might occur be
 See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for details.
 ====
 
-* Support for AMD and Intel 64-bit architectures (x86-64-v2).
+* Support for AMD, Intel 64-bit (x86-64-v2), and ARM64-based (`arm64` or `aarch64`) architectures.
```

Comment:
- s390x is supported, too
- The following items are only for x86-64 relevant, but I have no idea if there are similar requirements for the other architectures:
  * Support for AMD and Intel 64-bit architectures (x86-64-v2).
  * Support for Intel 64 or AMD64 CPU extensions.
  * Intel VT or AMD-V hardware virtualization extensions enabled.
  * NX (no execute) flag enabled.